### PR TITLE
Update lsi-raid ITL command

### PR DESCRIPTION
**Pull request by akrus**
Hi,

Currently lsi-raid command in ITL has only two arguments, but actually the check command allows much more to be provided. Apart from that, controller number & storcli path are not required arguments as they are pre-defined in check command already.

### CODE REVIEW

File: itl/plugins-contrib.d/raid-controller.conf [hunk 1]
```diff
@@ -31,16 +31,92 @@ object CheckCommand "lsi-raid" {
 
 	arguments = {
 		"-C" = {
-			required = true
 			value = "$lsi_controller_number$"
 			description = "Insert the controller number to be checked."
 		}
 		"-p" = {
-			required = true
```
Comment (dnsmichi) [inline itl/plugins-contrib.d/raid-controller.conf]: Why are those arguments removed as requirement?

File: itl/plugins-contrib.d/raid-controller.conf [hunk 2]
```diff
@@ -31,16 +31,92 @@ object CheckCommand "lsi-raid" {
...
+			set_if = "$lsi_nosudo$"
+			description = "Do not use sudo when running storcli."
+		}
+		"--nocleanlogs" = {
+			set_if = "$lsi_nocleanlogs$"
+			description = "Do not clean up the log files after executing storcli checks."
+		}
 	}
-
-	vars.storcli_path = "/usr/sbin/storcli"
```
Comment (dnsmichi) [inline itl/plugins-contrib.d/raid-controller.conf]: Having a default value here prevents to have that defined everywhere on the host/service level. Please revert this change.

Comment (akrus) [reply itl/plugins-contrib.d/raid-controller.conf]: Responded in next comment:
>> storcli_path is detected automatically by the script

File: itl/plugins-contrib.d/raid-controller.conf (hunk 1 above)
Comment (akrus) [reply itl/plugins-contrib.d/raid-controller.conf]: Responded in next comment.

**Comment by dnsmichi** [comment]
Fair enough, thanks.
